src/styles/components/_component-grid.scss

```diff
@@ -50,24 +109,93 @@ $breakpoint-tablet: 820px;
       ". max-scott .";
 
     &__item {
+      border: 3px solid black;
+      text-align: center;
+      align-items: center;
+
       &--accordion {
+        position: relative;
         grid-area: accordion;
+
+        &::before {
+          content: "";
+          position: absolute;
+          top: -11%;
+          left: -7%;
+          rotate: -15deg;
+          width: 40%;
+          height: 20%;
+          background-size: contain;
+          background-repeat: no-repeat;
+          background-image: url("../../../public/doodle_line.png");
+          z-index: 1;
+        }
       }
 
       &--dialog {
+        position: relative;
         grid-area: dialog;
+
+        &::before {
+          content: "";
+          position: absolute;
+          bottom: -3%;
+          right: -3%;
+          width: 10%;
+          height: 10%;
+          background-size: contain;
+          background-repeat: no-repeat;
+          background-image: url("../../../public/doodle_o.png");
+          z-index: 1;
+        }
       }
 
       &--disclosure {
+        position: relative;
         grid-area: disclosure;
+
+        &::before {
+          content: "";
+          position: absolute;
+          top: -5%;
+          right: -3%;
+          width: 20%;
+          height: 20%;
+          background-size: contain;
+          background-repeat: no-repeat;
+          background-image: url("../../../public/doodle_diamond.png");
+          z-index: 1;
+        }
       }
 
       &--tabs {
+        position: relative;
         grid-area: tabs;
+
+        &::before {
+          content: "";
+          position: absolute;
+          bottom: -15%;
+          left: -3%;
+          width: 30%;
+          height: 30%;
+          background-size: contain;
+          background-repeat: no-repeat;
+          background-image: url("../../../public/doodle_arrow.png");
+          z-index: 1;
+        }
       }
 
       &--max-scott {
         grid-area: max-scott;
+        padding: 0;
+        background: none;
+        border: none;
+
+        &::before,
+        &::after {
+          background-image: none;
```

Comment:
I might opt to use a `display: none` here to remove these images from the DOM when they aren't intended to be there as they are also creating some unintended space. The`background-image: none` will only remove the image from the element but not all the other properties that are still there.
![Untitled](https://user-images.githubusercontent.com/34315878/198408733-8afa4d6e-01dd-4281-97d6-ceba5c9b953b.png)